conmon/conmon.c

```diff
@@ -1388,7 +1393,9 @@ int main(int argc, char *argv[])
 	 * Glib does not support SIGCHLD so use SIGUSR1 with the same semantic.  We will
          * catch SIGCHLD and raise(SIGUSR1) in the signal handler.
 	 */
-	g_unix_signal_add (SIGUSR1, on_sigusr1_cb, pid_to_handler);
+	g_unix_signal_add(SIGUSR1, on_sigusr1_cb, pid_to_handler);
+	signal(SIGTERM, on_sig_exit);
+	signal(SIGINT, on_sig_exit);
```

Comment:
Does `conmon` actually get this?  I'm having trouble thinking of a keyboard-interrupt usecase.